dfdaemon/initializer/initializer.go

```diff
@@ -229,6 +229,13 @@ func initParam(options *options.Options) {
 		}
 	}
 
+	parsedTrustHosts := make(map[string]string)
```

Comment:
Did you test the more than two trust hosts through command parameters? If you did, please complete the description of this PR.

Reply:
@fengzixu  

Added more than two trust hosts and test it, please see the description of this PR.